### ISSUE TITLE
[doc] Elemental quickstart "Resources Needed" improvements

### ISSUE THREAD

**Issue by joesusecom** [suse-edge/suse-edge.github.io#183]
Resources Needed:

https://suse-edge.github.io/quickstart-elemental.html#id-resources-needed

Very detailed, which is good. But we need to agree on using the same terminology. This Quickstart says _"upstream" cluster_, the Metal³ one says "Management cluster".

Virtual machine: Other then testing/demos, is that really a use case for Elemental?

"resolvable hostname": For the cluster or controlled node?

I'm using Tumbleweed myself. :-)

But we should not recommend using it in an official Quickstart for a commercial offering.

**Comment by agracey** [comment]
Agreed but am unsure about what to put there instead. I don't want to make it look like you need to run the tooling on SLE

**Comment by ranjinimn** [comment]
@joesusecom can you please let me know if there is anything that I need to with this issue?
